WordPress/src/main/res/values/strings.xml

```diff
@@ -1533,7 +1533,10 @@
     <string name="error_media_load">Unable to load media</string>
     <string name="error_media_save">Unable to save media</string>
     <string name="error_media_canceled">Uploading media were canceled</string>
-    <string name="error_media_recover_post">Unable to upload this post\'s media</string>
+    <string name="error_media_recover_post">Could not upload media</string>
```

Comment:
⚠️ We're missing a period in the sentence. 🙂

Reply:
I didn't use a period here following https://material.io/design/communication/writing.html#principles (see "Skip unnecessary punctuation"). I'm not sure about it, feels a bit weird when you see 2 different messages.

@osullivanchris I'm not sure if we want to follow material design strictly here (or if we have WP guidelines for writing).

Reply:
Thank you for the link. It looks like you are right and it seems consistent with the other error messages above this.

Reply:
@osullivanchris, we'd love to hear what you think. 🙂

Reply:
Lets leave it without the period as all the other errors are the same for now. 

For future...
I checked our style guide and couldn't find something specific. The closest thing I saw was 
"Use periods at the end of list items that consist of complete sentences."
Not everyone would use periods in list items I imagine. So it seems like we lean on the side of using a period if that example is anything to go by.

Separately we could update them all to have periods if we think its better, but lets leave it for this issue. I think @jklausa might have been doing a similar inventory for iOS.